.github/workflows/phpstan/autoload.php

```diff
@@ -25,6 +25,17 @@ function requireFileIfItExists(string $filepath) : bool {
 require_once _PS_ROOT_DIR_ . '/install-dev/classes/exception.php';
 require_once _PS_ROOT_DIR_ . '/install-dev/classes/session.php';
 require_once _PS_ROOT_DIR_ . '/var/SymfonyRequirements.php';
+$moduleCronJobs = _PS_ROOT_DIR_ . '/modules/cronjobs/cronjobs.php';
+if (file_exists($moduleCronJobs)) {
+    require_once $moduleCronJobs;
+} else {
+    class CronJobs extends Module
+    {
+        public static function addOneShotTask($task, $description, $execution = array())
```

Comment:
From the outside it seems like a dirty fix for a bad code design 😅  . Is there no way we rather fix the bad code that needed you to do this instead of adding this trick ?

If you dont want to burden the scope of this PR, we can add a `@todo` for later 😉

Reply:
I agree

Reply:
Fixed